x-pack/plugins/infra/common/inventory_models/host/metrics/tsvb/host_network_traffic.ts

```diff
@@ -23,71 +23,35 @@ export const hostNetworkTraffic: TSVBMetricModelCreator = (
       id: 'tx',
       metrics: [
         {
-          field: 'system.network.out.bytes',
-          id: 'max-net-out',
-          type: 'max',
-        },
-        {
-          field: 'max-net-out',
-          id: 'deriv-max-net-out',
-          type: 'derivative',
-          unit: '1s',
-        },
-        {
-          id: 'posonly-deriv-max-net-out',
-          type: 'calculation',
-          variables: [{ id: 'var-rate', name: 'rate', field: 'deriv-max-net-out' }],
-          script: 'params.rate > 0.0 ? params.rate : 0.0',
-        },
-        {
-          function: 'sum',
-          id: 'seriesagg-sum',
-          type: 'series_agg',
+          field: 'host.network.egress.bytes',
+          id: 'avg-net-out',
+          type: 'avg',
         },
       ],
-      split_mode: 'terms',
-      terms_field: 'system.network.name',
+      split_mode: 'everything',
```

Comment:
What does this mean?

Reply:
it basically means `groupBy`.  when using the `system.network` fields we grouped by `system.network.name` which is the network interface name.  But now we don't do that now because the new field (host.network) returns the sum of all the interfaces and there is no such equivalent.